modules/api/php/endpoints/candidate/visit/electrophysiology/channels.class.inc

```diff
@@ -143,4 +143,4 @@ class Channels extends Endpoint implements \LORIS\Middleware\ETagCalculator
     {
         return md5(json_encode($this->_handleGET($request)->getBody()));
     }
-}
```

Comment:
I tried to remove this change but I can't. They seem to be exactly the same, I did not add a newline... same for the other similar changes in this PR